cypress/e2e/pages/articles/testsForCanonicalOnly.js

```diff
@@ -74,18 +74,16 @@ export default ({ service, pageType, variant = 'default' }) =>
     }
 
     if (liteEnabledServices.includes(service)) {
-      describe('Canonical Lite Site CTA', () => {
-        it.skip('should have a lite site CTA', () => {
-          cy.get('[data-e2e="to-lite-site"]').within(() => {
-            cy.get('a')
-              .should('have.attr', 'href')
-              .then($href => {
-                cy.get('a').click();
-                cy.url().should('contain', $href).should('contain', '.lite');
-              });
-          });
-          cy.go('back');
+      it.skip('should have a lite site link', () => {
```

Comment:
Does this still need skipped?

Reply:
Kinda yes - until #12663 is merged

Reply:
So I thinl I'll merge this first, and ☝️ after - lots of merge conflicts & code changes in the same place 🤦‍♀️